.ember-cli

```diff
@@ -1,9 +0,0 @@
-{
-  /**
-    Ember CLI sends analytics information by default. The data is completely
-    anonymous, but there are times when you might want to disable this behavior.
-
-    Setting `disableAnalytics` to true will prevent any data from being sent.
-  */
-  "disableAnalytics": false
-}
```

Comment:
I'm surprised that this doesn't show up as "moved". Did you explicitly delete it?

Reply:
fixed